src/pydata_sphinx_theme/assets/styles/base/_color.scss

```diff
@@ -152,143 +44,36 @@ html[data-theme="light"] {
 */
 html[data-theme="dark"] {
   /*****************************************************************************
-  * theme colors
+  * main colors
   */
 
-  // main colors
   --pst-color-primary: rgb(76, 145, 219);
   --pst-color-success: rgb(72, 135, 87);
   --pst-color-info: rgb(64, 125, 191);
```

Comment:
I think the admonition headers are a bit faded on dark screens, it is hard to see them, could we brighten them slightly or is that too complicated to do without making the configuration more complex?